core/src/main/java/org/apache/iceberg/CatalogProperties.java

```diff
@@ -48,6 +48,27 @@ private CatalogProperties() {}
 
   public static final boolean CACHE_CASE_SENSITIVE_DEFAULT = true;
 
+  /**
+   * Controls policy of caffeine cache
+   *
+   * <p>Behavior of specific values of cache.strategy:
+   *
+   * <ul>
+   *   <li>EXPIRE_AFTER_ACCESS - cache entries are never evicted as long as they are being accessed
+   *       frequently
+   *   <li>EXPIRE_AFTER_WRITE - cache entries are evicted frequently after cache write
```

Comment:
`cache entries are evicted frequently` I'm not sure what this means TBH. They are evicted after a configured interval. If it's configured short, then frequently, if not then not.